File: elasticsearch/es-client/src/main/java/me/kzv/elasticsearch/model/SubwayDocument.java
package me.kzv.elasticsearch.model;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class SubwayDocument {
  private String code;
  private String station;
  private String chosung;
  private String jamo;
  private String engtokor;
  private String line;
  private String excode;
}

File: elasticsearch/searching-server/src/main/kotlin/me/kzv/searchingserver/document/SubwayDocument.kt
package me.kzv.searchingserver.document

import org.springframework.data.annotation.Id
import org.springframework.data.elasticsearch.annotations.Document

@Document(indexName = "subway")
class SubwayDocument(
    @Id
    val id: String?,
    val station: String,
    val line: String,
    val excode: String
) {

}